### ISSUE TITLE
Error fetching models: Error: Failed to refresh access token.

### ISSUE THREAD

**Issue by kmrrakesh** [Azure-Samples/digital-twins-explorer#267]
#Error: Error fetching models: Error: Failed to refresh access token.
![image](https://user-images.githubusercontent.com/13743867/174598234-5b89283e-af5b-4a91-a487-225177450ca8.png)

##Code source
https://github.com/Azure-Samples/digital-twins-explorer

**Comment by jamiewilbraham** [comment]
Hi @kmrrakesh,

If you are running locally, have you tried stopping the server and logging in again via the "az login" command?

Thanks!

**Comment by kmrrakesh** [comment]
> Hi @kmrrakesh,
> 
> If you are running locally, have you tried stopping the server and logging in again via the "az login" command?
> 
> Thanks!

Yes @jamiewilbraham . I tried logging in via the "az login" command

**Comment by nnmer** [comment]
linking to #306
